pom.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-parent</artifactId>
+        <version>1.9.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-jdbc-sink</artifactId>
+    <name>Debezium JDBC Sink Connector</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <!-- Debezium parent -->
+        <version.debezium>${project.version}</version.debezium>
+        <version.mysql.driver>8.0.27</version.mysql.driver>
+        <version.postgres.driver>42.3.1</version.postgres.driver>
+        <version.hibernate>5.5.3.Final</version.hibernate>
```

Comment:
Is there a specific reason why we're based on Hibernate 5.5.3 rather than 5.6.3.Final?  In addition, has this been tested with any of the Hibernate 6 beta artifacts to see what, if any, changes will be necessary to upgrade in the near future?

Reply:
There's no specific reason about that. I just started with this version and we can upgrade it.

Reply:
Lets go ahead and roll it forward to the latest stable release.